### DISCUSSION TITLE
Uploads fails at 99%

### DISCUSSION THREAD

**Discussion by solidGad** [iterate-ch/cyberduck#15202] [Q&A]
Hello. I have been testing S3 apps with XNS and I have ran in an issue with Mountain Duck and CyberDuck where the upload fails at 99% every time. Seems to happen on any file over 1.5 GB. smaller files the issue doesn't happen. Anything over 1.5 GB i can see it syncing and then it gets to 99%, the sync restarts a few times and then it fails with Mountain Duck. 

With CyberDuck it gets to about the same point, 99%ish, then the transfer Window locks up and the upload will stay there forever until you close CyberDuck.

I did try a few other S3 apps that mount drives and didn't have the issue.

Thank you.

**Comment by dkocher** [comment]
Please let us know about the resolution.
